craco.config.cjs

```diff
@@ -100,10 +125,35 @@ module.exports = {
           return true
         })
 
-      // We're currently on Webpack 4.x which doesn't support the `exports` field in package.json.
-      // Instead, we need to manually map the import path to the correct exports path (eg dist or build folder).
-      // See https://github.com/webpack/webpack/issues/9509.
-      webpackConfig.resolve.alias['@uniswap/conedison'] = '@uniswap/conedison/dist'
+      webpackConfig.resolve.plugins = webpackConfig.resolve.plugins.map((plugin) => {
+        // Allow vanilla-extract in production builds.
+        // See https://sandroroth.com/blog/vanilla-extract-cra#production-build.
+        if (plugin instanceof ModuleScopePlugin) {
```

Comment:
Could you explain the relation between this and Webpack upgrade?

Reply:
I think that create-react-app 5 has additional restrictions to prevent bad practices, and this is one of them - it prevents you from including files in your build which shouldn't obviously be there.